b_lambda_layer_common/source/python/util/http_call.py

```diff
@@ -37,6 +37,15 @@ def call(method, url, fields=None, headers=None, **urlopen_kw) -> HTTPResponse:
         """
         http = urllib3.PoolManager()
 
+        logger.info(
```

Comment:
```suggestion
        logger.debug(
```
This information is not frequently needed and clutters log a lot.

Reply:
Agreed.